lldb/test/API/commands/frame/var-dil/basics/ArraySubscript/TestFrameVarDILArraySubscript.py

```diff
@@ -62,7 +62,7 @@ def test_subscript(self):
         self.expect(
             "frame var 'int_arr[-1]'",
             error=True,
-            substrs=["unrecognized token"],
+            substrs=["failed to parse integer constant"],
```

Comment:
This is more of the previous PR, but I think this should actually work. Current "frame var" will happily access a pointer/array with negative indexes.